src/FacultyDirectory/ClientApp/src/components/UserTable/UserInput.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Button, Input, InputGroup, InputGroupAddon } from 'reactstrap';
+
+export const UserInput = (props: any) => {
+    const [username, setUsername] = useState("");
+
+    const handleUsername = (e: any) => {
+        setUsername(e.target.value)
```

Comment:
It's worth loading the `prettier` and `eslint` extensions into vscode if they aren't there already.  We have config files already in the repo that should load the rulesets and take care of little stuff like forgetting a semicolon or changing between single and double-quotes